docs/source/install.rst

```diff
@@ -58,7 +58,11 @@ To create a new virtualenv for LiberTEM, you can use the following command:
 
 .. code-block:: shell
 
-    $ virtualenv -p python3.7 ~/libertem-venv/
+    $ virtualenv -p python3.x ~/libertem-venv/
+
+Replace :code:`x` with current Python version installed. To find out which
+version of Python is installed on your system run the :code:`python --version`
+or :code:`python -V` command.
```

Comment:
Hmm, this is kind of redundant. If you do `python --version` and put that into the `virtualenv` command, you could as well do `virtualenv -p python ~/libertem-venv/` (or even leave off the `-p` switch entirely).

Also note that the `python` binary can point to either Python 2 or Python 3 - this is detailed in [PEP 394](https://www.python.org/dev/peps/pep-0394/#for-python-runtime-distributors). So if I followed your instructions on my system, I would get a Python 2 virtualenv, which is not compatible with LiberTEM. To make sure you get Python 3, you can use the `python3` binary:

>We expect Unix-like software distributions (including systems like macOS and Cygwin) to install the python2 command into the default path whenever a version of the Python 2 interpreter is installed, and the same for python3 and the Python 3 interpreter

The point of using `-p python3.7` was to get a compatible Python version, not just any installed python3. So maybe we should use `-p python3` as the easiest way that works on most systems, but point out that you need to substitute a compatible version if you have multiple python versions installed. For example, I have Python 3.6, 3.7 and 3.8 installed on my system, so they would be all compatible, but if someone had Python 3.5 and 3.6 installed, they possibly would have to explicitly use `python3.6` to get LiberTEM running.

Note that the README.rst also contains virtualenv creation, so it should be changed there, too.

Reply:
Thanks! Such a wonderful explanation. Made some changes, hope that reflect what you mentioned.